liquigraph-core/src/main/java/org/liquigraph/core/io/lock/LockableConnection.java

```diff
@@ -77,6 +94,7 @@ public static LockableConnection acquire(Connection delegate, LiquigraphLock loc
     @Override
     public void close() throws SQLException {
         lock.release(this);
+        delegate.setAutoCommit(previousAutoCommit);
```

Comment:
not sure this is 100% safe (what happens if we change autocommit after commit?)
@mvitz @fpavageau WDYT?